io.openems.edge.evcs.wallbe/src/io/openems/edge/evcs/wallbe/api/Wallbe.java

```diff
@@ -0,0 +1,297 @@
+         * <ul>
+         * <li>Interface: Wallbe
+         * <li>Type: Integer
+         * <li>Unit: A
+         * </ul>
+         */
+        CURRENT_L2(Doc.of(OpenemsType.INTEGER).unit(Unit.AMPERE).accessMode(AccessMode.READ_ONLY)),
+        /**
+         * Current on Phase L3.
+         * <ul>
+         * <li>Interface: Wallbe
+         * <li>Type: Integer
+         * <li>Unit: A
+         * </ul>
+         */
+        CURRENT_L3(Doc.of(OpenemsType.INTEGER).unit(Unit.AMPERE).accessMode(AccessMode.READ_ONLY)),
+        /**
+         * Current Power being drawn.
+         * <ul>
+         * <li>Interface: Wallbe
+         * <li>Type: Integer
+         * <li>Unit: W
+         * </ul>
+         */
+        ACTIVE_POWER(Doc.of(OpenemsType.INTEGER).unit(Unit.WATT).accessMode(AccessMode.READ_ONLY)),
+        /**
+         * Current Energy Session.
+         * <ul>
+         * <li>Interface: Wallbe
+         * <li>Type: Integer
+         * <li>Unit: kWh
+         * </ul>
+         */
+        ENERGY(Doc.of(OpenemsType.INTEGER).unit(Unit.KILOWATT_HOURS).accessMode(AccessMode.READ_ONLY)),
+        ;
+        private final Doc doc;
+
+        ChannelId(Doc doc) {
+            this.doc = doc;
+        }
+
+        public Doc doc() {
+            return this.doc;
+        }
+
+    }
+
+    /**
+     * Gets the Channel for {@link ChannelId#MAXIMUM_CHARGE_CURRENT}.
+     *
+     * @return the Channel
+     */
+    default WriteChannel<Short> getMaximumChargeCurrentChannel() {
+        return this.channel(ChannelId.MAXIMUM_CHARGE_CURRENT);
+    }
+
+    /**
+     * Sets the Value of {@link ChannelId#MAXIMUM_CHARGE_CURRENT}.
+     *
+     * @param value the new value
+     */
+    default void setMaximumChargeCurrent(short value) throws OpenemsError.OpenemsNamedException {
+        WriteChannel<Short> channel = this.getMaximumChargeCurrentChannel();
+        channel.setNextWriteValue(value);
+    }
+
+    /**
+     * Gets the Channel for {@link ChannelId#WALLBE_STATUS}.
+     *
+     * @return the Channel
+     */
+    default Channel<String> getWallbeStatusChannel() {
+        return this.channel(ChannelId.WALLBE_STATUS);
+    }
+
+    /**
+     * Gets the Value of {@link ChannelId#WALLBE_STATUS}.
+     *
+     * @return the value
+     */
+    default String getWallbeStatus() {
+        Channel<String> channel = this.getWallbeStatusChannel();
+        return channel.value().orElse(channel.getNextValue().orElse("0"));
+    }
+
+    /**
+     * Gets the Channel for {@link ChannelId#CURRENT_L1}.
+     *
+     * @return the Channel
+     */
+    default Channel<Integer> getCurrentL1Channel() {
+        return this.channel(ChannelId.CURRENT_L1);
+    }
+
+    /**
+     * Gets the Value of {@link ChannelId#CURRENT_L1}.
+     *
+     * @return the value
+     */
+    default int getCurrentL1() {
+        Channel<Integer> channel = this.getCurrentL1Channel();
+        return channel.value().orElse(channel.getNextValue().orElse(0));
+    }
```

Comment:
I would omit the `orElse()`'s and change the return type to `Value<Integer>`, like it's done in other interfaces.
Maybe add `public` to make it clearer, and write it in one line `this.getCurrentL1Channel().value()`.
Same for the other getters. 
Those getters that are not needed (if there are any) somewhere, you can also leave out in my opinion.